core/ir/aarch64/codec.py

```diff
@@ -209,7 +209,7 @@ def indent_append(text):
                         indent_append('    ASSERT(0);')
                 enc_key = fallthrough_instr_id(
                     pattern.opcode, pattern.opcode_bits, pattern.opnd_bits)
-                if enc_key in FALLTHROUGH and patten.opndset == FALLTHROUGH[enc_key].opndset:
+                if enc_key in FALLTHROUGH and pattern.opndset == FALLTHROUGH[enc_key].opndset:
```

Comment:
@joshua-warburton (original author), @AssadHashmi -- there must be no tests that trigger this path?  Would it be straightforward to add such a test?

Reply:
Moved to top-level.